### ISSUE TITLE
Add query params for managing caching

### ISSUE THREAD

**Issue by mammuth** [mammuth/play-books-notes#5]

**Comment by brettvanderwerff** [comment]
Could you please add a description to this issue. I'd like to look closer at it to see if it is something I can help with, but I think there needs to be a bit more detail for me to see if it is something I can help with. If it was supposed to be self-explanatory, don't worry about it then. Thank you.

**Comment by mammuth** [comment]
currently the endpoints (/notes, /notes-api, /random-note) return the notes which are currently stored. However, it might be interesting to request the notes-api but update all notes before. This could be done via a request to eg. `/notes-api?cache=False` or `update=True` or something like that).

However, that's probably a rather unimportant feature I guess. Just a "nice to have".

**Comment by brettvanderwerff** [comment]
I see now. I'm taking a look at some extensions for managing cache in flask and will see what I can come up with.

**Comment by mammuth** [comment]
Nah, there's no need for an extension. With "caching" I just meant taking the notes from the in memory dictionary instead of updating the notes from Google drive.
